services/contact_us.js

```diff
@@ -2,13 +2,14 @@
 import aws from 'aws-sdk'; // eslint-disable-line import/no-unresolved, import/no-extraneous-dependencies
 import { validateAndSendEmail } from './ses';
 
-const contactUsEmailTo = process.env.CONTACT_US_EMAIL_TO;
-
-if (!contactUsEmailTo) {
-  throw new Error('CONTACT_US_EMAIL_TO environment variable not set!');
-}
 
 export default function doContactUs(event, cb) {
+  const contactUsEmailTo = process.env.CONTACT_US_EMAIL_TO;
+
+  if (!contactUsEmailTo) {
+    throw new Error('CONTACT_US_EMAIL_TO environment variable not set!');
+  }
```

Comment:
That's an unrelated issue. That needs to be fixed on the client- i.e. don't pass unverified content from the API to the client.